install-and-configure/advanced-configuration/resource-consumption.md

```diff
@@ -1,54 +1,99 @@
 
-Lowering query concurrency for the Kubecost ETL build will mean ETL takes longer to build, but consumes less memory. The default value is: `5`. This can be adjusted with the [Helm value](https://github.com/kubecost/cost-analyzer-helm-chart/blob/v1.93.2/cost-analyzer/values.yaml#L272):
+```yaml
+kubecostModel:
+  etlResolutionSeconds: 600
+```
 
-`--set kubecostModel.maxQueryConcurrency=1`
+**Soft memory limit.**  Optionally enabling impactful memory thresholds can ensure the Go runtime garbage collector throttles at more aggressive frequencies at or approaching the soft limit. There is not a one-size fits all value here, and users looking to tune the parameters should be aware that lower values may reduce overall performance if setting the value too low. If users set the the `resources.requests` memory values appropriately, using the same value for `softMemoryLimit` will instruct the Go runtime to keep its heap acquisition and release within the same bounds as the expectations of the pod memory use. More info on this environment variable can be found in [A Guide to the Go Garbage Collector](https://tip.golang.org/doc/gc-guide).
 
-## Lower query duration
+```yaml
+kubecostModel:
+  softMemoryLimit: 10GiB
+```
 
-Lowering query duration results in Kubecost querying for smaller windows when building ETL data. This can lead to slower ETL build times, but lower memory peaks because of the smaller datasets. The default values is: `1440` This can be tuned with the [Helm value](https://github.com/kubecost/cost-analyzer-helm-chart/blob/fa0b00de5a186e658ccb66792bcdc3b77c4170e9/cost-analyzer/templates/cost-analyzer-deployment-template.yaml#L817):
+## Aggregator
 
-`--set kubecostModel.maxPrometheusQueryDurationMinutes=300`
+Please refer to the [Aggregator docs](/install-and-configure/install/multi-cluster/federated-etl/aggregator.md) for more detaild configuration and troubleshooting information.
 
-## Lower query resolution
+**Lower retention period.** How much data to ingest from the federated store bucket, and how much data to keep in the DB before rolling the data off. Reducing these numbers will reduce load on aggregator. By default these are set to `91` and `49` respectively.
 
-Lowering query resolution will reduce memory consumption but will cause short running pods to be sampled and rounded to the nearest interval for their runtime. The default value is: `300s`. This can be tuned with the Helm value:
+{% hint style="warning" %}
+`etlHourlyStoreDurationHours` can be set to `0` if hourly cost granularity is not required in your environment. Note, that some features on the Savings page default to using hourly data, but can be toggled to use daily data.
+{% endhint %}
 
-`--set kubecostModel.etlResolutionSeconds=600`
+```yaml
+kubecostAggregator:
+  etlDailyStoreDurationDays: 61
+  etlHourlyStoreDurationHours: 0
+```
```

Comment:
@MikeFindsThings @srpomeroy @cliffcolvin @jessegoodier Curious what you all think about the recommendation for zero `etlHourlyStoreDurationHours`. I've asked some users to do it and it has turned out ok. I've also noted some limitations of this method in the doc.